docs/UserGuide.md

```diff
@@ -287,7 +294,7 @@ Format: `delete-wedding w/WEDDINGNAME [f/]`
 * The no contacts should be assigned to the `Wedding` before it is deleted.
 * If there are still contacts assigned, you can use `f/` to force the deletion of the `Wedding` and unassign all contacts.
 
-### Task Features
+## Task Features
 
 ### Creating a Task : `create-task`
 
```

Comment:
I think we will need to specify the different task types and their respective commands/ keywords